storage/tianmu/core/tianmu_table.cpp

```diff
@@ -1013,15 +1013,16 @@ int TianmuTable::binlog_insert2load_block(std::vector<loader::ValueCache> &vcs,
         case common::ColumnType::SMALLINT:
         case common::ColumnType::INT:
         case common::ColumnType::MEDIUMINT:
-        case common::ColumnType::BIGINT: {
+        case common::ColumnType::BIGINT:
+        case common::ColumnType::BIT: {
           types::BString s;
           int64_t v = *(int64_t *)(vcs[att].GetDataBytesPointer(i));
           if (v == common::NULL_VALUE_64)
             s = types::BString();
           else {
-            types::TianmuNum rcd(v, m_attrs[att]->Type().GetScale(), m_attrs[att]->Type().IsFloat(),
-                                 m_attrs[att]->TypeName());
-            s = rcd.ToBString();
+            types::TianmuNum tianmu_d(v, m_attrs[att]->Type().GetScale(), m_attrs[att]->Type().IsFloat(),
```

Comment:
and here, why var is named with `_d` ?

Reply:
This is replaced rcd, I made a change from `rcd` to `tianmu_d`.